.goreleaser.yml

```diff
@@ -33,6 +33,13 @@ builds:
     ldflags:
       - -s
       - -w
+      - -X 'k8s.io/component-base/version.buildDate={{ .CommitDate }}'
+      - -X 'k8s.io/component-base/version.gitCommit={{ .FullCommit }}'
+      - -X 'k8s.io/component-base/version.gitTreeState={{ .Env.GIT_TREE_STATE }}'
```

Comment:
Is this env var defined?

Reply:
Yes https://github.com/search?q=repo%3Ad2iq-labs%2Fcapi-runtime-extensions%20GIT_TREE_STATE&type=code